tests/integration/cairo1/hint_locals/start_warp/data/proxy/lib.cairo

```diff
@@ -0,0 +1,25 @@
+use starknet::syscalls::call_contract_syscall;
+use array::ArrayTrait;
+use starknet::SyscallResult;
+use starknet::contract_address::ContractAddress;
+use array::SpanTrait;
+
+#[contract]
+mod ProxyContract {
+    use starknet::syscalls::call_contract_syscall;
+    use array::ArrayTrait;
+    use starknet::SyscallResult;
+    use starknet::contract_address::ContractAddress;
+    use array::SpanTrait;
+
+    #[view]
+    fn check_remote_timestamp(at: ContractAddress) -> felt252 {
+        let res = call_contract_syscall(
+            at,
+            1629366970795695773070250980946704058198029661891218027632744717191545197319,
```

Comment:
Could you add a comment naming the selector used here?

Reply:
[Add an explaining comment for selector](https://github.com/software-mansion/protostar/pull/1867/commits/4f506c8638b3f56a57dca08bf94f063f697b3919)